environment.yml

```diff
@@ -20,4 +20,4 @@ dependencies:
       - git+https://github.com/parsl/parsl.git  # TODO (wardlt) 23Apr22: Remove when Parsl is pushed to PyPi next
       - pydantic
       - proxystore>=0.3.1
-      - funcx<=0.3.6
+      - funcx<=1.0.7
```

Comment:
Why `>=`? Should we not assume this will work for future versions of FuncX?

Reply:
I see that our `setup.py` requirements point to funcX>1. Should we also change that to FuncX>=1.0.7?

Reply:
Good question. Why was it previously set to <=? I suspect pinning to >= 1 or 1.0.7 would be best?

Reply:
I figured the `<=` was either me messing it up, or to avoid some weird bug in pre-v1 FuncX.

Reply:
I've updated it to require >=1.0.5, which is when funcX added rabbitmq-backed executors.